source/PrintDiamond.cpp

```diff
@@ -0,0 +1,86 @@
+#include "PrintDiamond.h"
+
+int answer()
+{
+	return 6 * 9;
+}
+
+void PrintDiamondClass::InitArray()
+{
+	m_alphabet = { { 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 
+					 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z' } };
+	for (int i = 0; i <= 25; i++)
+	{
+		m_numbers[i] = i + 1;
```

Comment:
The way I see it, you could get rid of m_numbers entirely and just assign `i+1` to `m_loop_control` in line 32.